src/common/path_util.cpp

```diff
@@ -82,11 +82,22 @@ static std::filesystem::path GetBundleParentDirectory() {
 
 static auto UserPaths = [] {
 #ifdef __APPLE__
-    std::filesystem::current_path(GetBundleParentDirectory());
+    // Start by assuming the base directory is the bundle's parent directory.
+    std::filesystem::path base_dir = GetBundleParentDirectory();
+    std::filesystem::path user_dir = base_dir / "user";
```

Comment:
Should use the existing `PORTABLE_DIR` constant.

Reply:
Done, thanks for pointing out! :D

Reply:
@squidbus Using `PORTABLE_DIR` results in shadPS4 ignoring everything after, creates user folder near the executable no matter the scenario.

Reply:
`PORTABLE_DIR` is literally just `"user"`, you should be able to replace that part and it would be no different.

So: `std::filesystem::path user_dir = base_dir / PORTABLE_DIR;`